tests/bluetooth/shell/nrf5.conf

```diff
@@ -0,0 +1,27 @@
+CONFIG_BT=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_ATT_PREPARE_COUNT=2
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_CONSOLE_SHELL=y
+CONFIG_BT_SHELL=y
+CONFIG_BT_DEVICE_NAME="test shell"
+CONFIG_BT_L2CAP_TX_BUF_COUNT=6
+
+# For host max. throughput
+CONFIG_BT_RX_BUF_LEN=258
+CONFIG_BT_L2CAP_TX_MTU=247
+CONFIG_BT_ATT_TX_MAX=6
```

Comment:
I guess this is the maximum ATT throughput, btw don't we need a more aggressive connection parameter in order to attain better speeds?

Reply:
Not with DLE I reckon, since then longer CIs allow for longer LL data packets. @cvinayak can confirm what the ideal one is